src/openrct2/rct1/RCT1.h

```diff
@@ -539,11 +539,11 @@ namespace RCT1
             uint8_t StaffOrders;         // 0xC6
             uint8_t PeepIsLostCountdown; // 0xC6
         };
-        RCT12RideId Photo1RideRef;      // 0xC7
-        uint32_t PeepFlags;             // 0xC8
-        rct12_xyzd8 PathfindGoal;       // 0xCC
-        rct12_xyzd8 PathfindHistory[4]; // 0xD0
-        uint8_t NoActionFrameNum;       // 0xE0
+        RCT12RideId Photo1RideRef;    // 0xC7
+        uint32_t PeepFlags;           // 0xC8
+        RCT2xyzd8 PathfindGoal;       // 0xCC
```

Comment:
`RCT2xyzd8` is not a proper name, it should be `RCT12xyzd8`, as it is evidently used in RCT1 structures.